packages/gen-ai/docs/developer/README.md

```diff
@@ -0,0 +1,21 @@
+# Developer Documentation
+
+This directory contains documentation for developers contributing to the Gen AI system.
+
+## Available Guides
+
+- [BFF Testing Guide](../../bff/README.md) - Comprehensive guide for testing the BFF component
+
+## Additional Resources
+
+- [Architectural Decision Records](../adr/) - ADRs documenting key architectural decisions
+- [API Documentation](../bff/openapi/) - OpenAPI specifications for the BFF
```

Comment:
There's no `docs/bff/openapi` path. -> https://github.com/opendatahub-io/odh-dashboard/tree/main/packages/gen-ai/docs/bff/openapi 

I think we want something like `../../bff/openapi/` -> https://github.com/opendatahub-io/odh-dashboard/tree/main/packages/gen-ai/bff/openapi/